server/errorHandler.js

```diff
@@ -0,0 +1,16 @@
+const logger = require('../log.js')
+
+const production = process.env.NODE_ENV === 'production'
```

Comment:
Is there a reason we're only removing the stack trace from the UI in `production`? I think for both  development and prod purposes we can log the error in to the server, and remove the additional code of checking the enviroment in this handler

Reply:
They do that in other projects such as Pathfinder and Use Of Force, so I just followed the same pattern here.

Reply:
all of our code should behave the same across all environments and locally. I'll raise a ticket to remove the rest of these switches, but these ones can be tackled in this PR.